### ISSUE TITLE
join bottom pieces better

### ISSUE THREAD

**Issue by jaredjennings** [jaredjennings/dactyl-keyboard#51]
So I made the bottom pieces bigger [7da24ba], since I don't have to stop printing, clean up and ride home cause I got my own printer. But the waves at their joints are too small to sand, and when I do, too much of them is removed. They need to be bigger. The reason I made them smaller is to obtain more precise alignment of the pieces, particularly vertically. But little imperfections in the print spoil the alignment.

So bigger waves. Would rotating them work better for vertical alignment? Or maybe they should be polar? A more complicated wave than a sine? Maybe I should cut irregularities with a little cutting bit, instead of trying to sand them? I need a good way to find them first. Like when you go to the dentist, they do something to your teeth, and then they check your bite with carbon paper and grind down here and there till all your teeth fit together just right again. Except my plastic is presently black. Powder of some kind? Flour?

**Comment by jaredjennings** [comment]
Making the waves gentler just let pieces shear against each other in the z axis without anything to stop them. I'm using Sci-Grip 4 now, which solvent-welds the pieces; I'm not sure whether it softens them where applied, but it ought to. Also filing everything is annoying and doesn't always make it fit better. Sometimes there are gaps, even without having filed much material off. The Fourier box https://gitlab.com/jaredjennings/fourier-box has this problem, but I spent 30 hours printing it so I'm just going to make it work somehow. Probably with epoxy.

**Comment by jaredjennings** [comment]
The way I solved it with the Fourier box was with friction welding. I clamped the pieces together, friction welded the entire seam on one side, friction welded a dotted line on the other side, and squirted Sci-Grip 4 in using a little bottle with a syringe tip. I got too happy with the clamp on one joint, and the resulting box failed to be flat. It still works, but it's not that great.

So I'm not still doing a pieced bottom, because I'm not doing an extra column anymore, so fingers crossed the whole bottom fits on my printer. My leading hypothesis about how to proceed with piecing is to stop trying to make the pieces contact each other across the whole joint, and instead make some sort of alignment jig that fits into the pieces in small places, like a big asterisk whose points are small balls that fit each into dimples on the model. *handwave* This then is what puts the pieces in the right position and orientation relative to each other, rather than the surfaces of the joint. Then friction weld the joint, to fill in plastic, then Sci-Grip it for good measure. Oh - 3D pens are also a thing - I saw a guy who matched two broad flat curved surfaces by scribbling across the joint with a 3D pen. I don't have one of those.

I haven't tried any of that above, so it might be another six-month boondoggle. As long as it's not in the way of making a keyboard, I'm not doing it yet.
